maintainers/maintainer-list.nix

```diff
@@ -5540,4 +5540,9 @@
     github = "tesq0";
     name = "Mikolaj Galkowski";
 	};
+  thiagokokada = {
```

Comment:
It would be nice if you added the maintainership as a separate commit so they are independent of each other.

Reply:
Also, this list is supposed to be in alphabetical order, so move your entry upwards. I can see that some haven't followed this, but fix your entry at least :-)

Reply:
Does the same apply to all-packages.nix?

Reply:
That one is harder to keep sorted since it's huge, but at least there are attempts. I usually do a dirlist and place it close to the package before or after in the dirlist of where I put it. In this case close to `bb` or `bibletime` or something like that...